jenkins/ci.suse.de/cloud-socok8s.yaml

```diff
@@ -5,5 +5,14 @@
     # the credentialId for opensusegithubapi
     repo-credentials: c2350527-476a-45df-b406-84f028614682
     jobs:
-      - '{name}-integration'
+      - '{name}-integration-{tempest}':
+          tempest: 'without-tempest'
+          run_tempest: false
+          tempest_test_type: smoke
```

Comment:
(nit) This seems redundant, or is the purpose to run smoke tests as a minimum?

Reply:
I was thinking of not running tempest with every PR change as it adds quite substantially to total job runtime.

More details are available in this discussion on socok8s PR
https://github.com/SUSE-Cloud/socok8s/pull/644#discussion_r309836402

Let me know what you think ?

Reply:
Right, outside of the tempest test type (which I think we can discuss on the other PR), this line seems redundant when `run_tempest` is set to `False`.

Reply:
Okay, got it. You are asking for removing tempest_test_type line when run_tempest is false. Yes, I can remove that as it was added for consistency sake but does not have any logic implication so can be removed.